components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -9,6 +9,21 @@ type BatchCheckRequest struct {
 	Config Config   `json:"config"`
 }
 
+type MockServerFromBatchServiceResponse struct {
+	Host       string `json:"host"`
+	Protocol   string `json:"protocol"`
+	Port       int    `json:"port"`
+	Error      error  `json:"error"`
+	StatusCode int    `json:"status_code"`
+}
+
+type MockServerRequestBody struct {
```

Comment:
check if we can use original mock server api models instead of creating new

Reply:
Done